package.json

```diff
@@ -3,9 +3,14 @@
   "private": true,
   "description": "A collection of components that make it easy to build interactive charts with D3",
   "scripts": {
-    "postinstall": "lerna bootstrap",
-    "publish": "lerna publish --conventional-commits --message \"chore: publish\" --yes",
-    "test": "lerna run test --stream --concurrency 1"
+    "postinstall": "./node_modules/.bin/lerna bootstrap",
+    "publish": "./node_modules/.bin/lerna publish --conventional-commits --message \"chore: publish\" --yes",
+    "bundle": "./node_modules/.bin/lerna run bundle --stream --concurrency 1",
+    "pretest": "./node_modules/.bin/lerna run bundle --stream --concurrency 1",
```

Comment:
If this is always going to be the same as the `bundle` script, would it make sense to have `"pretest": "npm run bundle"` ?